2016-11-03 14:19:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Sequence "ID" not found; SQL statement:
ALTER TABLE movies ALTER COLUMN id RESTART WITH 1 [90036-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.dml.AlterSequence.setColumn(AlterSequence.java:59)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5649)
	at org.h2.command.Parser.parseAlter(Parser.java:4843)
	at org.h2.command.Parser.parsePrepared(Parser.java:344)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.persistence.Initializer.eraseData(Initializer.java:32)
	at gr.aueb.mscis.sample.persistence.Initializer.prepareData(Initializer.java:43)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2016-11-03 14:19:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Sequence "ID" not found; SQL statement:
ALTER TABLE movies ALTER COLUMN id RESTART WITH 1 [90036-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.dml.AlterSequence.setColumn(AlterSequence.java:59)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5649)
	at org.h2.command.Parser.parseAlter(Parser.java:4843)
	at org.h2.command.Parser.parsePrepared(Parser.java:344)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.persistence.Initializer.eraseData(Initializer.java:32)
	at gr.aueb.mscis.sample.persistence.Initializer.prepareData(Initializer.java:43)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2016-11-03 14:19:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Sequence "ID" not found; SQL statement:
ALTER TABLE movies ALTER COLUMN id RESTART WITH 1 [90036-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.dml.AlterSequence.setColumn(AlterSequence.java:59)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5649)
	at org.h2.command.Parser.parseAlter(Parser.java:4843)
	at org.h2.command.Parser.parsePrepared(Parser.java:344)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.persistence.Initializer.eraseData(Initializer.java:32)
	at gr.aueb.mscis.sample.persistence.Initializer.prepareData(Initializer.java:43)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2016-11-03 14:19:04 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Sequence "ID" not found; SQL statement:
ALTER TABLE movies ALTER COLUMN id RESTART WITH 1 [90036-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.dml.AlterSequence.setColumn(AlterSequence.java:59)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5649)
	at org.h2.command.Parser.parseAlter(Parser.java:4843)
	at org.h2.command.Parser.parsePrepared(Parser.java:344)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.persistence.Initializer.eraseData(Initializer.java:32)
	at gr.aueb.mscis.sample.persistence.Initializer.prepareData(Initializer.java:43)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2016-11-03 14:19:04 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Sequence "ID" not found; SQL statement:
ALTER TABLE movies ALTER COLUMN id RESTART WITH 1 [90036-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.dml.AlterSequence.setColumn(AlterSequence.java:59)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5649)
	at org.h2.command.Parser.parseAlter(Parser.java:4843)
	at org.h2.command.Parser.parsePrepared(Parser.java:344)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.persistence.Initializer.eraseData(Initializer.java:32)
	at gr.aueb.mscis.sample.persistence.Initializer.prepareData(Initializer.java:43)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2016-11-03 14:26:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into movies values ('snowden', 2012, 'test') [21002-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2016-11-03 14:27:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO MOVIES VALUES ID[*] (NULL, 'snowden', 2012, 'test') "; expected "("; SQL statement:
insert into movies values ID (null, 'snowden', 2012, 'test') [42001-192]
2016-11-03 14:27:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'test' (MOVIES: YEAR INTEGER NOT NULL)"; SQL statement:
insert into movies values (null, 'snowden', 2012, 'test') -- (NULL, 'snowden', 2012, 'test') [22018-192]
2016-11-03 14:28:09 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:847)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:874)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:58)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getEntityManagerFactory(JPAUtil.java:16)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getCurrentEntityManager(JPAUtil.java:25)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 77 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 72 more
2016-11-03 14:28:09 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:847)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:874)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:58)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getEntityManagerFactory(JPAUtil.java:16)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getCurrentEntityManager(JPAUtil.java:25)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 77 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 72 more
2016-11-03 14:28:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:847)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:874)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:58)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getEntityManagerFactory(JPAUtil.java:16)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getCurrentEntityManager(JPAUtil.java:25)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 77 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 72 more
2016-11-03 14:28:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:847)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:874)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:58)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getEntityManagerFactory(JPAUtil.java:16)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getCurrentEntityManager(JPAUtil.java:25)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 77 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 72 more
2016-11-03 14:28:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:847)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:874)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:58)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getEntityManagerFactory(JPAUtil.java:16)
	at gr.aueb.mscis.sample.persistence.JPAUtil.getCurrentEntityManager(JPAUtil.java:25)
	at gr.aueb.mscis.sample.service.MovieServiceTest.setup(MovieServiceTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 77 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/bzafiris/workspace-aueb/sample/test.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 72 more
2020-02-26 23:53:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:53:58 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:55:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:55:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-26 23:55:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:31 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:33 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:35 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:37 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:40 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:42 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:44 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:46 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:48 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-26 23:55:50 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 01:05:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "JPACLASS" not found; SQL statement:
select MarketsData from JpaClass MarketsData order by MarketsData.MDID desc  [42102-192]
2020-02-27 01:09:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "M" not found; SQL statement:
select m from MarketsData m order by m.MDID desc  [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:852)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.service.ShowMarketServiceTest.test_getOfflineMarketImage(ShowMarketServiceTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2020-02-27 01:11:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "M" not found; SQL statement:
select m from MarketsData order by m.MDID desc  [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:852)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.service.ShowMarketServiceTest.test_getOfflineMarketImage(ShowMarketServiceTest.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2020-02-27 01:12:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "M" not found; SQL statement:
select m from MarketsData m order by m.MDID desc  [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:852)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.service.ShowMarketServiceTest.test_getOfflineMarketImage(ShowMarketServiceTest.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2020-02-27 01:12:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "M" not found; SQL statement:
select m from MarketsData m [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:852)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:290)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:87)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:78)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:188)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1505)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:263)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1485)
	at gr.aueb.mscis.sample.service.ShowMarketServiceTest.test_getOfflineMarketImage(ShowMarketServiceTest.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2020-02-27 21:38:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:38:58 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:39:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:39:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:39:51 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:39:53 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:39:55 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:39:57 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:39:59 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:01 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:03 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:05 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:07 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:09 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:11 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:40:13 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:41:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:41:22 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:24 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:26 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:28 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:30 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:32 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:34 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:36 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:38 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:40 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:42 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:41:44 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:44:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:44:35 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 21:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:57 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:58 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:58 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:58 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:58 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:47:58 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:57:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:57:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 21:57:34 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:36 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:38 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:40 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:42 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:44 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:46 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:48 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:50 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:52 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:54 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 21:57:56 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 22:01:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMERS" not found; SQL statement:
delete from Customers [42102-192]
2020-02-27 22:01:24 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:26 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:28 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:30 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:32 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:34 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:36 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:38 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:43 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:45 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:01:47 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:03:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:03:16 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:18 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:20 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:22 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:24 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:26 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:28 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:30 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:32 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:34 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:36 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:03:38 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Timeout trying to lock table ; SQL statement:
delete from MarketsData [50200-192]
2020-02-27 22:08:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:08:46 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:01 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:02 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:02 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:02 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:02 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:15:02 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:43 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:43 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:24:44 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:31 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:31 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:31 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:28:32 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:29:14 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[17]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[18]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 22:40:38 jdbc[19]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[17]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[18]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
2020-02-27 23:00:00 jdbc[19]: exception
org.h2.jdbc.JdbcSQLException: Table "TRANSACTION" not found; SQL statement:
delete from Transaction [42102-192]
